.circleci/config.yml

```diff
@@ -404,6 +404,46 @@ jobs:
           no_output_time: "3m0s"
           <<: *check_scion_connections
 
+      - run:
+          name: NEW -- Check updated configuration matches VPN setup
+          working_directory: ~/project/.circleci
+          command: |
+            # Check configuration on AP
+            echo "Client AP 1405 new VPN configuration"
+            docker-compose exec -T as1405 /bin/bash -c \
+              "apt-get install -y jq > /dev/null 2>&1; "\
+              "jq '"'.BorderRouters."br20-ffaa_0_1405-2".Interfaces'"' /etc/scion/gen/ISD20/ASffaa_0_1405/endhost/topology.json; "\
+              "ip address show tun0; "\
+              "ip address show tun0 | "'grep $(jq '"'"'.BorderRouters."br20-ffaa_0_1405-2".Interfaces["3"].PublicOverlay.Addr'"'"' /etc/scion/gen/ISD20/ASffaa_0_1405/endhost/topology.json --raw-output)/'
```

Comment:
The quoting gives me fever. Can you just use `'` for the entire line?

Maybe you could also simplify the queries a bit, e.g. `jq ".BorderRouters[].Interfaces[].PublicOverlay.Addr"` will return all addresses.

Reply:
Cleaned up the quotes mess for that one.